themes/package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "remnote-ctp-port-personal",
+	"version": "1.0.0",
+	"main": "index",
+	"repository": "https://github.com/coldenate/remnote.git",
+	"author": "Nathan Solis <66754842+coldenate@users.noreply.github.com>",
```

Comment:
Also I don't know if people are able to email that email. 
Actually let me try out

Reply:
True. the package JSON I'll edit to make sure it's proper, but currently right now I'm just using it to track the dependencies. Overall I don't do it for anything else except tracking the dependencies.

Reply:
I guess you didn't get my lovely email :(
Well good to know

Reply:
Aww :(
👍